packages/loot-core/src/server/sync/index.ts

```diff
@@ -243,7 +243,7 @@ type Message = {
   dataset: string;
   old?: unknown;
   row: string;
-  timestamp: number;
+  timestamp: Timestamp;
```

Comment:
I updated this based on the failing typecheck in the test but it seems like it's the test which is incorrect. I'll revert this change and update the test.